sql-plugin/src/main/330+/scala/org/apache/spark/sql/execution/datasources/parquet/ShimCurrentBatchIterator.scala

```diff
@@ -71,6 +72,8 @@ class ShimCurrentBatchIterator(
   config.setBoolean(SQLConf.PARQUET_BINARY_AS_STRING.key, false)
   config.setBoolean(SQLConf.PARQUET_INT96_AS_TIMESTAMP.key, false)
   config.setBoolean(SQLConf.CASE_SENSITIVE.key, false)
+  // timestamp_NTZ flag is introduced in Spark 3.4.0
+  ParquetTimestampNTZShims.setupTimestampNTZConfig(config, conf)
```

Comment:
I am not certain if this is correct thing to do. This is in 330+ shim but depending on whether the Spark version is 3.3 or 3.4, it's NOOP or config is set. We already have this distinction for Spark-3.3 -[ here](https://github.com/NVIDIA/spark-rapids/blob/801a339bb3088131110076d8f3385ad9a57761f5/sql-plugin/src/main/311until340-all/scala/com/nvidia/spark/rapids/shims/ParquetTimestampNTZShims.scala#L23) and 3.4 - [here](https://github.com/NVIDIA/spark-rapids/blob/bfdecc3714ab71f76e42686d9a9d193824ae762c/sql-plugin/src/main/340%2B/scala/com/nvidia/spark/rapids/shims/ParquetTimestampNTZShims.scala#L23).
So I didn't create another object in 330until340/340+ which would call corresponding `ParquetTimestampNTZShims.setupTimestampNTZConfig` . If this code is confusing, I can split up as mentioned before.

Reply:
LGTM